prometheus-ksonnet/grafana/grr.libsonnet

```diff
@@ -0,0 +1,48 @@
+/*
+ * The `grr` function below will render Grafana resources generated by this
+ * Grafana library (e.g. as `grafanaDashboards`) into Kubernetes style objects,
+ * with kind/apiVersion. (Grizzly)[https://github.com/grafana/grizzly)
+ * has the ability to consume resources in this format.
+ *
+ * It is invoked with a different entrypoint and handed your jsonnet as input. E.g
+ * if your main entrypoint is `main.jsonnet`, this can be invoked like so:
+ *
+ * grr.jsonnet:
+ * ```
+ * local grafana = import 'prometheus-ksonnet/grafana/grr.libsonnet';
+ * grafana.grr(import 'main.jsonnet')
+ */
+local grafanaAPI = 'grafana.com/v1';
```

Comment:
Does this version mean something? I'd refer to who is intended to consume it, for example `grizzly.dev/v1alpha1` or something, in line with `tanka.dev/v1alpha1`?

Reply:
given we don't own grizzly.dev, I suggest grafana.com/grizzly/v1alpha1